### ISSUE TITLE
Issue in starting development env and signing up

### ISSUE THREAD

**Issue by Rupeshiya** [thedevs-network/kutt#484]
**Issues:**
1. On running npm run dev (on windows NODE_ENV is not recognised)
2. Server is sending status of 500 on signup without any info.

**Screenshots:**
![image](https://user-images.githubusercontent.com/31209617/123070234-36095780-d431-11eb-87c9-3a46ee1c000e.png)
![image](https://user-images.githubusercontent.com/31209617/123070373-520cf900-d431-11eb-9006-f012d0a175eb.png)

**Expected outcomes:**
1. Environment commands should be setup on the basis of the OS to make it compatible. (NODE_ENV in linux os, SET NODE_ENV for windows)
2. Proper logging mechanism should be there to check the error that server is throwing.

Should I proceed to fix these issues @poeti8 ?

**Comment by poeti8** [comment]
[cross-env](https://www.npmjs.com/package/cross-env) seems like a good tool to solve this issue. 

>Should I proceed to fix these issues @poeti8 ?

Sure, why not.

**Comment by hashlash** [comment]
For the second issue, perhaps it was the same case with https://github.com/thedevs-network/kutt/issues/269#issuecomment-628604256?

**Comment by kengher** [comment]
> 

Yes, most likely the same case. Going through [#269](https://github.com/thedevs-network/kutt/issues/269#issuecomment-628604256) and fixing my own SMTP credentials fixed it for me.
